include/timeutils.h

```diff
@@ -88,4 +90,13 @@ int strtime_short(const time_t *t, struct timeval *now, int flags, char *buf, si
 extern time_t timegm(struct tm *tm);
 #endif
 
+static inline char *xctime(time_t *t)
+{
+	char *buf;
+
+	buf = xcalloc(1, 64);
+	ctime_r(t, buf);
+	return buf;
+}
+
```

Comment:
Not sure if I like it... would be better to use in code directly ctime_t and static buffer?